std/concurrency.d

```diff
@@ -1661,14 +1710,49 @@ void yield(T)(T value)
             {
                 tid.send(e);
             }
+
```

Comment:
This isn't a show stopper but there's no reason to clutter the diff with extraneous whitespace.